### ISSUE TITLE
Feature Concept: Migrant Rehabilitator

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#19518]
Idea: First time we see a new ckey/player age 0, we send them a popup to refer them to more information to help them fit in better based on where they have experience. This could help if we ever do a recruiting drive or go to the hub.

`Welcome to /vg/. It looks like you're playing here for the first time. Please select your past experience with SS13.`
* I've never played before

Links to the getting started wiki page (which could be spruced up)

* I've played on low-RP servers such as Hippie or TG

Very basic information about escalation and information like "please do not use chatspeak such as LOL, JK, OMG while in character"

* I've played on high-RP servers such as Polaris, Bay, or Aurora

A warning that on /vg/, fights can happen without much warning. Maybe some basic tips on robustitude.

* I've played on a server which does not use the "Secret" game type, like Colonial Marines

Link to an explanation about how we handle antagonists and round types.

* I've played here before!

Closes the window without further harassment

**Comment by Reviire** [comment]
What if we gave brand new players a special role? So other players know they're new, and ones who care can try to help them without needing to be informed first. 

Their first 10 rounds they start as a high vis assistant.

**Comment by Kurfursten** [comment]
As much as I'd love to see a "Work Placement Migrant" thing as a real role, which I think would be hilarious, I honestly believe the best way to learn SS13 the first time is through a simple job like cargo. So restricting them to assistant could leave them in an "idle hands" situation.

**Comment by DeityLink** [comment]
Agreed, assistant is good to explore the station like a tourist, but not to learn the ropes of being a non-antag like you'll be 90% of your games.

Great jobs for new players include:

=Cargo=
* Learn to use computers
* Great for players looking for a more practical-oriented introduction to the game
* Handling crates
* Deal with players of many different jobs and find many different items

=Bartender=
* Basically an introduction to Chemistry
* Great for players looking for a more RP-oriented introduction to the game
* You get to make requests to botanists and chefs which gives you opportunity for inter-job teamwork
* People will come in the bar to break stuff (or build without your permission), or harm Pun-Pun, and you're allowed to deal with them to some extent, which gives you an introduction to robustness (and also gun handling if the beanbags come into play)

=Xenobiology=
* Basically a minigame, but that can give advantages to your department or the crew, or nice tools with which to perform your antag objectives
* An introduction to Science as a whole
* An introduction to dealing with NPC mobs in a "relatively" safe environment, compared to the asteroid

Other jobs are either
* too complex for beginners (Chemist, Genetics, Xenoarch, most head roles (oh who am I kidding))
* dangerous for the crew if you fuck it up (Engineering, Atmospherics, Virology)
* dangerous for yourself (Miner, Anomalist, Clown, Mime)
* boring if you're not able yet to perform some basic RP (IAA, Chaplain, Detective)

**Comment by Orangerush** [comment]
Was this related to the anonymous imageboard thread about giving pop ups instead of bans? I like the idea though

**Comment by Exxion** [comment]
>I've played here before!

No further harassment but maybe a message to admins

**Comment by Kurfursten** [comment]
@Orangerush It's well documented that coders and admins do not ever visit 4chan

**Comment by DeityLink** [comment]
I stand for exception though.

**Comment by Exxion** [comment]
He was joking. A bunch do.

**Comment by Kurfursten** [comment]
Go easy on Deity, they don't have sarcasm in France

**Comment by DeityLink** [comment]
My sarcasm detector has been defective my entire life ;_;
